lib/internal/crypto/keys.js

```diff
@@ -356,21 +415,107 @@ function isKeyObject(key) {
   return key instanceof KeyObject;
 }
 
+// Our implementation of CryptoKey is a simple wrapper around a KeyObject
+// that adapts it to the standard interface. This implementation also
+// extends the JSTransferable class, allowing the CryptoKey to be cloned
+// to Workers.
+// TODO(@jasnell): Embedder environments like electron may have issues
+// here similar to other things like URL. A chromium provided CryptoKey
+// will not be recognized as a Node.js CryptoKey, and vice versa. It
+// would be fantastic if we could find a way of making those interop.
+class CryptoKey extends JSTransferable {
+  constructor(
+    keyObject,
+    algorithm,
+    keyUsages,
+    extractable) {
+    super();
+    this[kKeyObject] = keyObject;
+    this[kAlgorithm] = algorithm;
+    this[kExtractable] = extractable;
+    this[kKeyUsages] = keyUsages;
```

Comment:
why are these public symbol properties instead of private fields?

Reply:
The performance penalty is still far too much at the moment.

Reply:
Gotcha. Perhaps a comment to that effect would be helpful?